Notifications.md

```diff
@@ -0,0 +1,58 @@
+subscribed to characteristic 00001624-1212-efde-1623-785feabcd123
+
+received:
+
+* `0f 00 04 01 01 25 00 00 00 00 10 00 00 00 10` - ?
+* `0f 00 04 02 01 26 00 00 00 00 10 00 00 00 10` - ?
+* `0f 00 04 37 01 27 00 00 00 00 10 00 00 00 10` - ?
+* `0f 00 04 38 01 27 00 00 00 00 10 00 00 00 10` - ?
+* `09 00 04 39 02 27 00 37 38` - Port Group: 0x39 consists of 0x37 and 0x38
+* `0f 00 04 32 01 17 00 00 00 00 10 00 00 00 10` - ?
+* `0f 00 04 3a 01 28 00 00 00 00 10 00 00 00 02` - ?
+* `0f 00 04 3b 01 15 00 02 00 00 00 02 00 00 00` - ?
+* `0f 00 04 3c 01 14 00 02 00 00 00 02 00 00 00` - ?
+* `05 00 82 37 01` - Motor A (0x37) starts movement
+* `05 00 82 37 0a` - Motor A (0x37) stops movement
+* `05 00 82 37 05` - Motor A (0x37) Is reported when a command is received while the motor is still in movement
+* `05 00 82 32 0a` - Reports a change of the LED state
+
+* Byte 0 as always: message length
+
+* Byte 1? Protocol version? HiByte of message length? 
```

Comment:
Not related to notifications, but I just noticed that Jorge is using 0x1 for Byte 1 in his motor shell example:

	gatttool -i hci1 -b 00:16:53:A4:CD:7E --char-write-req --handle=0x0e \
           --value=0c018139110a80099B64647f03

Don't know why :-)

Reply:
That's what I captured. Have to find time to look at it again.